app/api/api_v1/endpoints/report.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import json
+
+from fastapi import APIRouter, Depends, Response, HTTPException
+from fpdf import FPDF
+from sqlalchemy.orm import Session
+
+import crud
+import utils
+from api import deps
+from models import User
+from schemas import FarmProfile, MachineryAssetsOfFarm, PlotParcelDetail, ReportDB, ReportCreate, Message, ReportDBID
```

Comment:
Are these schemas used?

Reply:
Some are, removed the ones that weren't being used